packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.js

```diff
@@ -58,10 +59,14 @@ const defaultConfig = {
  * @public
  */
 const MoonstoneDecorator = hoc(defaultConfig, (config, Wrapped) => {
-	const {ri, i18n, spotlight, float, noAutoFocus, overlay, textSize, skin, highContrast} = config;
+	const {ri, i18n, spotlight, float, noAutoFocus, overlay,
+		textSize, skin, highContrast, disableFullscreen} = config;
```

Comment:
We should add `disableFullscreen` to the `defaultConfig` object above as well. `defaultConfig` acts as the "interface" for the HOC configuration and is the home for documentation (even though it's lacking in this case) so we should keep it in sync with the config options used.